btk/obs_conditions.py

```diff
@@ -3,18 +3,76 @@
 import numpy as np
 import galsim
 import descwl
+from collections import namedtuple
 
+# A simple class archetype to serve as a dictionary without having to write the field names every time
+Survey = namedtuple("Survey", ["name", "pixel_scale", "psf_scale", "bands", "mean_sky_level", "exp_time", "zero_point"])
 
-all_surveys = {
-    "LSST": {
-        "name": "LSST",
-        "bands": ("y", "z", "i", "r", "g", "u"),
-        "pixel_scale": 0.2,
-    },
-    "DES": {"name": "DES", "bands": ("i", "r", "g", "z"), "pixel_scale": 0.263},
-    "CFHT": {"name": "CFHT", "bands": ("i", "r"), "pixel_scale": 0.185},
-    "HSC": {"name": "HSC", "bands": ("y", "z", "i", "r", "g"), "pixel_scale": 0.17},
-}
+pix_ROMAN = 0.11
+pix_RUBIN = 0.2
+pix_HST = 0.06
+pix_EUCLID = 0.101
+pix_HSC = 0.167
+pix_DES = 0.263
+pix_CFHT = 0.185
+
+# Sigma of the psf profile in arcseconds.
+sigma_ROMAN = 0.11 * np.array([1.68, 1.69, 1.86, 2.12, 2.44, 2.71])  # https://arxiv.org/pdf/1702.01747.pdf Z-band
+sigma_RUBIN = np.array([0.327, 0.31, 0.297, 0.285, 0.276, 0.267])  # https://www.lsst.org/about/camera/features
+sigma_EUCLID = np.array([0.16])  # https://sci.esa.int/documents/33859/36320/1567253682555-Euclid_presentation_Paris_1Dec2009.pdf
+sigma_HST = np.array([0.074])  # Source https://hst-docs.stsci.edu/display/WFC3IHB/6.6+UVIS+Optical+Performance#id-6.6UVISOpticalPerformance-6.6.1 800nm
+sigma_HSC = np.array([0.306, 0.285, 0.238, 0.268, 0.272])  # https://hsc-release.mtk.nao.ac.jp/doc/ deep+udeep
+
+
+Euclid = Survey("Euclid",
+                pix_EUCLID,
+                sigma_EUCLID,
+                ["VIS"],
+                np.array([22.9]),
+                np.array([2260]),
+                np.array([6.85]))
+HST = Survey("HST",
+             pix_HST,
+             sigma_HST,
+             ["f814w"],
+             np.array([22]),
+             np.array([3000]),
+             np.array([20]))
+HSC = Survey("HSC",
+             pix_HSC,
+             sigma_HSC, ["g", "r", "i", "z", "y"],
+             np.array([21.4, 20.6, 19.7, 18.3, 17.9]),
+             np.array([600, 600, 1200, 1200, 1200]),
+             np.array([91.11, 87.74, 69.80, 29.56, 21.53]))
+Roman = Survey("Roman",
+               pix_ROMAN,
+               sigma_ROMAN,
+               ["F062", "Z087", "Y106", "J129", "H158", "F184"],
+               np.array([22, 22, 22, 22, 22, 22]), ## Not Checked!!!
+               np.array([3000, 3000, 3000, 3000, 3000, 3000]),  ## Not Checked!!!
```

Comment:
maybe make this an issue so we don't forget that these are not checked?

Reply:
you could also raise a warning whenever this survey is used somewhere in the DrawBlendsGenerator like 
```
if survey.name == 'Roman': 
    warnings.warn("Values for mean_sky_level and exp_time for the Roman survey might be incorrect")
```

Reply:
I have been looking into it but it seems like there are different possible exposing times ; it does seem to me that those values are placeholders so we should definitely look into it at some point.

Reply:
It's still early for Roman, so I think it's fine for now.

Reply:
I agree. I will write an issue to make sure we tackle this later like Ismael said.